tests/test_TaskMetadata.py

```diff
@@ -43,10 +43,7 @@
     "MTOPDomainClassification",
     "MTOPIntentClassification",
     "NordicLangClassification",
-    "ScalaDaClassification",
-    "ScalaNbClassification",
-    "ScalaNnClassification",
-    "ScalaSvClassification",
+    "MultiScalaClassification",
```

Comment:
```suggestion
```

Reply:
we don't want to test metadata at all for this? or you meant to replace by `ScalaClassification` give the suggested change above?

Reply:
This is historic datasets which are not checked. This new dataset should be checked.